feature/mpls/sr/otg_tests/srte_egress_node_forwarding/egress_node_forwading_test.go

```diff
@@ -0,0 +1,674 @@
+	ipHdrV6.Dst().SetValue(ateTrafficDstIPv6)
+
+	tcpHdrV6 := flowV4.Packet().Add().Tcp()
+	tcpHdrV6.SrcPort().SetValue(10000 + (packetsPerFlow % 50000)) // pseudo-random src port > 1024
+	tcpHdrV6.DstPort().SetValue(tcpDstPort)
+
+	udpHdrV6 := flowV6.Packet().Add().Udp()
+	udpHdrV6.SrcPort().SetValue(20000 + (packetsPerFlow % 40000)) // pseudo-random src port > 1024
+	udpHdrV6.DstPort().SetValue(udpDstPort)
+
+	return otgConfig
+}
+
+func enableCapture(t *testing.T, config gosnappi.Config, port string) {
+	config.Captures().Clear()
+	t.Log("Enabling capture on ", port)
+	config.Captures().Add().SetName(port).SetPortNames([]string{port}).SetFormat(gosnappi.CaptureFormat.PCAP)
+
+}
+
+func startCapture(t *testing.T, otg *otg.OTG) gosnappi.ControlState {
+	t.Helper()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.START)
+	otg.SetControlState(t, cs)
+
+	return cs
+}
+
+func stopCapture(t *testing.T, otg *otg.OTG, cs gosnappi.ControlState) {
+	t.Helper()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.STOP)
+	otg.SetControlState(t, cs)
+}
+
+func processCapture(t *testing.T, otg *otg.OTG, port string) string {
+	bytes := otg.GetCapture(t, gosnappi.NewCaptureRequest().SetPortName(port))
+	time.Sleep(30 * time.Second)
+	capture, err := os.CreateTemp("", "pcap")
+	if err != nil {
+		t.Errorf("ERROR: Could not create temporary pcap file: %v\n", err)
+	}
+	if _, err := capture.Write(bytes); err != nil {
+		t.Errorf("ERROR: Could not write bytes to pcap file: %v\n", err)
+	}
+	defer capture.Close()
+
+	return capture.Name()
+}
+
+func validatePackets(t *testing.T, filename string, protocolType string) error {
+
+	mplsPacketCount := int32(0)
+
+	handle, err := pcap.OpenOffline(filename)
+	if err != nil {
+		return err
+	}
+	defer handle.Close()
+
+	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
+
+	for packet := range packetSource.Packets() {
+		if protocolType == "ipv4" {
+			if ipLayer := packet.Layer(layers.LayerTypeIPv4); ipLayer != nil {
+				ip, _ := ipLayer.(*layers.IPv4)
+				if ip.DstIP.Equal(net.ParseIP(ateTrafficDstIPv4)) {
+					if mplsLayer := packet.Layer(layers.LayerTypeMPLS); mplsLayer != nil {
+						mplsPacketCount += 1
+					}
+				}
+				if ip.TTL == mplsOuterTTL-1 {
+					t.Logf("TTL decremented by 1")
+					break
+				} else {
+					return fmt.Errorf("Didin't get TTL as expected. Expected: %v, Got: %v", mplsOuterTTL-1, ip.TTL)
+				}
+			}
+		} else {
+			if ipLayer := packet.Layer(layers.LayerTypeIPv6); ipLayer != nil {
+				ip, _ := ipLayer.(*layers.IPv6)
+				if ip.DstIP.Equal(net.ParseIP(ateTrafficDstIPv6)) {
+					if mplsLayer := packet.Layer(layers.LayerTypeMPLS); mplsLayer != nil {
+						mplsPacketCount += 1
+					}
+				}
+				if ip.HopLimit == mplsOuterTTL-1 {
+					t.Logf("TTL decremented by 1")
+					break
+				} else {
+					return fmt.Errorf("Didin't get TTL as expected. Expected: %v, Got: %v", mplsOuterTTL-1, ip.HopLimit)
```

Comment:
https://google.github.io/styleguide/go/decisions#error-strings + 
https://google.github.io/styleguide/go/decisions#got-before-want

```suggestion
					return fmt.Errorf("unexpected TTL - got: %v, want: %v", ip.HopLimit,  mplsOuterTTL-1)
```